lib/xlog/shorten.go

```diff
@@ -58,6 +59,10 @@ func (c GCLClient) Shorten(ctx context.Context, url string) (short string, err e
 		return "", trace.Wrap(err, "reading response from URL shortener")
 	}
 
+	if resp.StatusCode != 200 {
+		return "", trace.Wrap(fmt.Errorf("%v returned: %v, body: %q", shortenerEndpoint, resp.Status, data))
```

Comment:
Is this just so we have error logs in the test?
nit: `trace.Errorf` would have the same effect w/o the need to use an intermediary (`fmt.Errorf`)

Reply:
fmt.Errorf cleaned up in 893bf8b.

> Is this just so we have error logs in the test?

Yes. These tests aren't run by `make test` (they're nondeterministic integration tests that require cloud auth) and the shortening of code seems dead.  The difference is illustrated here:

```
walt@gcp:~/git/robotest$ go test -count 1 ./lib/xlog/                                                                                                                                   [141]

level=error msg="structured error test" errorf="there was an error <nil>" multi_wrap="<nil>" where="[/lib/xlog/all_test.go:44]" wrapped_no_message="<nil>" wrapped_with_message="<nil>"
--- FAIL: TestShorten (0.19s)
    all_test.go:66: unable to shorten url: https://www.googleapis.com/urlshortener/v1/url returned: 404 Not Found, body: "Not Found"
FAIL
FAIL    github.com/gravitational/robotest/lib/xlog      0.697s
FAIL
walt@gcp:~/git/robotest$ git checkout 05bccb91d26df554f71de25f739bb11df88de3c8     
walt@gcp:~/git/robotest$ go test -count 1 ./lib/xlog/                         

level=error msg="structured error test" errorf="there was an error <nil>" multi_wrap="<nil>" where="[/lib/xlog/all_test.go:44]" wrapped_no_message="<nil>" wrapped_with_message="<nil>"
--- FAIL: TestShorten (0.17s)
    all_test.go:66: unable to shorten url: Decoding response "Not Found"
                invalid character 'N' looking for beginning of value
FAIL
FAIL    github.com/gravitational/robotest/lib/xlog      0.664s
FAIL
```

I'd remove/fix all the shorten stuff, but that was orthogonal to the goal of this PR, and not a priority at this moment.